### PR TITLE
testing creating files

### LINKED ISSUE #7: Testing creating checklists
## Class Checklist
- [x] Create an Issue
- [x] Create a Branch
- [ ] Make commits
- [ ] Open a pull request
- [ ] Collaborate!
- [ ] Merge pull request
- [ ] Have a great time!

### PR DESCRIPTION
**Pull request by amiguptapro**
Fixes #7

### CODE REVIEW

File: githubclass/epping.md [hunk 1]
```diff
@@ -0,0 +1 @@
+testing testing
```
Comment (githubteacher) [inline githubclass/epping.md:1]: @amiguptapro please add some markdown

```md
# hello
world
```

**Review by githubteacher — CHANGES_REQUESTED** [review]

**Review by allenc23 — APPROVED** [review]
looks good!

File: githubclass/epping.md (hunk 1 above)
Comment (amiguptapro) [reply githubclass/epping.md:1]: sure. Adding...

**Review by githubteacher — APPROVED** [review]
LGTM :shipit:

**Pull request merged**